src/common/serac_types.hpp

```diff
@@ -11,10 +11,7 @@
 
 // Option bundling enums
 
-enum class OutputType {
-  GLVis,
-  VisIt
-};
+enum class OutputType { GLVis, VisIt };
```

Comment:
Dont like this.

Reply:
It seems to just be whether or not the definition can fit on a single line or not.

Reply:
Apparently to get around this you have to add a comment to every line. sigh...

enum Mode {
  NONE,         // Comments
  SPLIT_FILES,  // On
  SINGLE_FILE,  // Every
  INVALID       // Line
};

Reply:
You could locally deactivate clang format.

Reply:
there's a few simple workarounds:
- a single comment inside the enum (don't need all lines, but you can if you want)
- change the linewidth setting
- locally disable formatting with
```
// clang-format off
    void    unformatted_code  ;
// clang-format on
```

Reply:
Thats better.

Reply:
I don't like the line width option for obvious reasons... just turning off clang-format is going to be helpful in a lot of places.

Reply:
Obvious but maybe still of personal taste?
Instead of using it in many places, maybe then changing the rule in clang-format config is the best option.

Reply:
I'll clarify.. Some people online suggest setting the line width to 0 so it doesn't do this.  Unless I read wrong, that line width option is not just for enums for the entire code.

On a similar note, stay with 120 length.

Reply:
Per [clang-format doc](https://clang.llvm.org/docs/ClangFormatStyleOptions.html):
bool AfterEnum Wrap enum definitions.
```
true:
enum X : int
{
  B
};

false:
enum X : int { B };
```

Reply:
Definitely agree the line width = 0 wouldn’t help!